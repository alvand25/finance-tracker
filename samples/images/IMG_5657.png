version https://git-lfs.github.com/spec/v1
oid sha256:78c98ed1e4908c0410fc8fa8bd79a7a16726a1075dc9e9569dd7dd9c4c063ca0
size 13073824
