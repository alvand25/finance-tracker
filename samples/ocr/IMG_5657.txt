\
nn
5e 5s = as. LE
' Te pes a ee
96-02 Qucena B1vd.
5unnysidz, HY 11104
(710) 706-6563
CELERY BUNCH 1.99 F
Regu1ar Price 2.69 . You 5aved 0.70
TAX 0.00
seit BALANCE 1.99
HHI IIE EKER RK EK KKK
VI5A Card - CONTACTLE55
; ACCOUNT NUMBER: **X4X#8%xxeXx7107
APPROVAL CODE: 072890
5EQUENCE NUMBER: 3211
TERMINAL ID:
TOTAL AIOUNT: $1.99 Purchase
RE5PON5E CODE: APPROVED
04/11/25 12:50pm 431 3
Petro 5e htt Pees ee ke cee 2 5e rae ee ces
VI5A 1,99
CHANGE 0.00
TOTAL NUMBER OF ITEM5 5OLD = J
By 5hoeaing ot eu Food wou coved:
an 1 neent Pee Teonn. nom