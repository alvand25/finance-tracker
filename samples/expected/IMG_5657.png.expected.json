{
    "store": "Key_Food",
    "items": [
        {
            "description": "CILANTRO MACHO",
            "price": 2.49
        },
        {
            "description": "BASIL",
            "price": 2.50
        }
    ],
    "subtotal": 4.99,
    "tax": 0.0,
    "total": 4.99,
    "payment_method": "CASH",
    "timestamp": "2024-04-11T16:49:00",
    "check_items_exactly": true
}